### ISSUE TITLE
ERROR in _this.options.callback is not a function while changing from 1.2.0->1.3.0

### ISSUE THREAD

**Issue by code-masala** [react-loadable/revised#2]
@tranvansang

**Comment by tranvansang** [comment]
@code-masala 

there were breaking changes in v1.3.0.

In v1.3.0, I rewrite the document and refine some interfaces.

You need to provide the callback to the webpack plugin.

Please use the implementation in the readme.

https://github.com/react-loadable/revised/tree/124734ad779de797a213e2a02983941790b8af5e#webpack-config

**Comment by tranvansang** [comment]
The reason is that exporting a new asset (or updating) after webpack freezes assets is not recommended.

That is why the webpack plugin was changed to require a callback that handles the output manifest.

**Comment by code-masala** [comment]
`	new ReactLoadablePlugin({
			filename: "react-loadable.json"
		}),
`
Bro can you tell me how this is affecting because this is causing the error

**Comment by tranvansang** [comment]
@code-masala 

Change to this


```javascript
const {writeFile} = require('fs/promises')

new ReactLoadablePlugin({
	async callback(manifest) {
		// save the manifest somewhere to be read by the server
		await writeFile(path.join(__dirname, 'react-loadable.json'), JSON.stringify(manifest, null, 2))
	},
})
```

**Comment by code-masala** [comment]
@tranvansang  now it is giving `ERROR in regeneratorRuntime is not defined`

**Comment by tranvansang** [comment]
How about this?

```javascript
const {writeFileSync} = require('fs')

new ReactLoadablePlugin({
	callback(manifest) {
		// save the manifest somewhere to be read by the server
		writeFileSync(path.join(__dirname, 'react-loadable.json'), JSON.stringify(manifest, null, 2))
	},
})
```

**Comment by tranvansang** [comment]
While this is totally not related to your current issue. I have just published v1.3.1 to handle the async error rejected by the callback. Tell me if you have any further error with this version.

**Comment by code-masala** [comment]
now error has gone.Working perfectly
